rdmo/core/templatetags/core_tags.py

```diff
@@ -132,3 +132,9 @@ def startswith(text, starts):
     if isinstance(text, str):
         return text.startswith(starts)
     return False
+
+
+@register.filter()
+@stringfilter
+def clean(string):
```

Comment:
Can here not django's builtin tag `striptags` be used instead?   
* https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#striptags
* [def strip_tags(value):](https://github.com/django/django/blob/994dc6d8a1bae717baa236b65e11cf91ce181c53/django/utils/html.py#L211)

Reply:
Yes, first we need to run markdown since otherwise we would have the markdown markup, then we remove everything again (but the `[<>&"\']`) which are removed by clean.